component/pom.xml

```diff
@@ -113,8 +113,14 @@
         <dependency>
             <groupId>org.wso2.extension.siddhi.map.json</groupId>
             <artifactId>siddhi-map-json</artifactId>
-            <version>${json.mapper.version}</version>
-            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.extension.siddhi.map.binary</groupId>
+            <artifactId>siddhi-map-binary</artifactId>
```

Comment:
Why you are not using <scope>test</scope> for binary, json and keyvalue mappers. They are only used in test cases, right ?